utils/remote_control/navigator_keyboard_control/nodes/navigator_keyboard_server.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+
+'''
+Keyboard Server: The keyboard server enables a client to control NaviGator
+using a remote keyboard. It runs a service that responds to KeyboardControl
+service requests. It manages the currently locked client through the use of
+UUIDs. Only keys received from the client with the locked UUID will be
+executed.
+'''
+
+
+import curses
+import uuid
+
+from navigator_msgs.srv import KeyboardControl
+from remote_control_lib import RemoteControl
+import rospy
+
+
+__author__ = "Anthony Olive"
+__maintainer__ = "Anthony Olive"
+__email__ = "anthony@iris-systems.net"
+__copyright__ = "Copyright 2016, MIL"
+__license__ = "MIT"
+
+
+rospy.init_node('keyboard_server')
+
+
+class KeyboardServer(object):
+
+    def __init__(self):
+        self.force_scale = rospy.get_param("/joystick_wrench/force_scale", 600)
+        self.torque_scale = rospy.get_param("/joystick_wrench/torque_scale", 500)
+
+        self.remote = RemoteControl('keyboard', "/wrench/keyboard")
+        rospy.Service('/keyboard_control', KeyboardControl, self.key_recieved)
```

Comment:
It seems like services aren't the way to do this. What motivated you to use a service instead of a topic